RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptions/ManageSubscriptionsViewModel.swift

```diff
@@ -53,6 +55,9 @@ final class ManageSubscriptionsViewModel: ObservableObject {
     @Published
     var inAppBrowserURL: IdentifiableURL?
 
+    @Published
```

Comment:
nit: can we skip this `Published`? It's not getting reloaded anywhere

Reply:
Yes! Removed in b82e80fd04f8ba240d2e72a8385286116c7bd42f